sql/src/main/java/io/crate/action/job/TransportJobAction.java

```diff
@@ -68,6 +69,20 @@ public TransportJobAction(TransportService transportService,
                 new NodeActionRequestHandler<JobRequest, JobResponse>(this) { });
     }
 
+    public void execute(String node, final JobRequest request, final ActionListener<JobResponse> listener, long timeout) {
```

Comment:
Can you change it do use TimeValue instead of long? That way the unit is explicit